libnmstate/varlink/README.varlink.md

```diff
@@ -0,0 +1,155 @@
+# Varlink support for libnmstate
+Varlink protocol ([Varlink.org](https://varlink.org/)) can used to communicate with libnmstate via stdin/out and varlink client implementation ([Existing Language binding](https://varlink.org/Language-Bindings#how-to-test-new-language-bindings)). Varlink protocol encodes all messages as JSON object and communicates over unix and tcp socket connections. Nmstate service is defined in io.nmstate interface address and libnmstate functions should called in as specified in varlink interface defination file `io.nmstate.varlink`. This implementation supports python3-varlink version 29.0.0 and unix socket connection only.
+
+
+## Initiate nmstate varlink service
+User can initate nmstate-varlink services using ```nmstatectl``` command-line tool by defining the unix socket address path. The current nmstate-varlink implementation limits only using unix sockets connection.
+
+unix socket connection
+```bash
+$ nmstatectl varlink /run/nmstate.so &
+```
+
+Nmstate-varlink systemd services default listening address is ```unix:/run/nmstate/nmstate.so```
+systemd service initiation
+```bash
+$ systemctl start nmstate-varlink.service
+```
```

Comment:
Please move all the information about the systemd support to the systemd implementation PR.

Reply:
Sorry for that. I have fix it